fragments/home/banner.yml

```diff
@@ -1,3 +1,8 @@
+# visible: if set to true, the banner will become visible on the homepage.
+# type: One of the following - info, success, warning, error
+# title: The title of the banner that will be visible to the user
+# content: The actual content of the banner. Because of the format of .yml files, this should start and end with a single quote so that you can write on multiple lines.
```

Comment:
maybe put an example of this in the comments? It might not be clear what you're describing here.

Reply:
I added a whole banner example below. Do you think that works okay?